astropy/utils/data.py

```diff
@@ -578,12 +578,14 @@ def get_pkg_data_fileobj(data_name, package=None, encoding=None, cache=True):
         file-like object's ``read`` method will return `str` (``unicode``)
         objects, decoded from binary using the given encoding.
 
-    cache : bool
+    cache : bool or "update", optional
```

Comment:
I think this is doc update oversight back in the days. `cache` all get funneled through the same low-level function, and should all mean the same throughout this file.